float32[] left_gaze_vector
float32[] right_gaze_vector
float32[] poses_T
float32[] poses_R
float32[] left_landmarks
float32[] right_landmarks
float32[] action_units
